test/special_flags/add2_extras.json

```diff
@@ -1,3 +1,4 @@
 {
+  "ignoreReuse": true,
```

Comment:
this flag used to be here but was removed, but `add2` is supposed to be the test for `ignoreReuse`
https://github.com/dnanexus/dxCompiler/blob/85c60e4ede211441fcb8c93c7103f1dd6f190d92/scripts/run_tests.py#L263-L267

@Gvaihir I am thinking about checking the job description to see if these flags are working. But would that be something you already suggested in refactoring the integration tests?

Reply:
I haven implemented any specific class of tests. They all will be subclasses of one parent class, extendable to any scenario. Main goal - whatever test subclass you add it will be easy to test that your addition works. So for now go as you plan